source/localizable/tutorial/subroutes.md

```diff
@@ -0,0 +1,398 @@
+For Super Rentals, we want to be able to display all rentals and allow the user to view additional information for each.
```

Comment:
How about "In addition to displaying a list of rentals, we want to show detailed information for individual rentals"